wurst/systems/core/ClassSelector.wurst

```diff
@@ -134,21 +132,11 @@ public class ClassSelector
             ..add(UNIT_SCOUT)
             ..add(UNIT_GATHERER)
 
-    private function initializePickers()
-        let pos = spawn.getCenter()
-        for member in tribe.getMembers()
-            pickers.push(createUnit(member, UNIT_HERO_PICKER, pos.polarOffset(angle(PI), 1200), angle(0)))
-            member.panCamToTimed(totem, 0)
-            doAfter(0.3) ->
-                member.selectSingle(totem)
```

Comment:
Why remove the selection of the totem? I thought that was kind of nice

Reply:
good catch, forgot to add it back

Reply:
idk why there's doAfter() here but it doesn't seem needed so I removed